Makefile

```diff
@@ -143,5 +156,6 @@ help:
 	$(call help_once,CARGO_OPTS          --  Changes `cargo` default behavior (e.g. --verbose).)
 	$(call help_once,NOVENDOR            --  Avoid to self detect and consume the vendored headers)
 	$(call help_once,                        shipped with libbpf-sys.)
+	$(call help_once,COV                 --  enable code coverage for testing.)
```

Comment:
We should write something like:

`requires llvm-cov and preferably rustup toolchain`

also something like:

`applies only to the target "test" ...`